client/my-sites/site-settings/transfer-site-to-user/site-transfer-warning.jsx

```diff
@@ -0,0 +1,98 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteDomain } from 'state/sites/selectors';
+import userSettings from 'lib/user-settings';
+
+class SiteTransferWarning extends PureComponent {
+	static propTypes = {
+		selectedSiteDomain: PropTypes.string,
+		currentUserEmail: PropTypes.string,
+		onAcknowledged: PropTypes.func,
+		translate: PropTypes.func,
+	};
+
+	render() {
+		const translate = this.props.translate;
+		const transOpts = {
+			args: {
+				domain: this.props.selectedSiteDomain,
+				email: this.props.currentUserEmail,
+			},
+		};
+
+		return (
+			<ActionPanel>
+				<ActionPanelBody>
+					<ActionPanelTitle>{ translate( 'Transfer Your Site' ) }</ActionPanelTitle>
+					<p>
+						{ translate(
+							'Transferring a site cannot be undone. Please read the following actions that will ' +
+								'take place when you transfer this site:'
+						) }
+					</p>
+					<ul>
+						<li>{ translate( 'You will be removed as owner of %(domain)s', transOpts ) }</li>
+						<li>
+							{ translate(
+								'You will not be able to access %(domain)s unless allowed by the new owner',
+								transOpts
+							) }
+						</li>
+						<li>
+							{ translate(
+								'Your posts on %(domain)s will be transferred to the new owner and ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Your posts on %s will be transferred to the new owner and will no longer be authored by your account.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).